### PR TITLE
Add Tailwind

### PR DESCRIPTION
**Pull request by brunvez**
Add Tailwind with https://github.com/rails/tailwindcss-rails

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import * as ActiveStorage from '@rails/activestorage';
 import '@hotwired/turbo-rails';
 import '../channels';
 import '../controllers';
+require('stylesheets/application.scss');
```
Comment (santib) [inline app/javascript/packs/application.js:11]: maybe use `import` ?

File: Gemfile [hunk 2]
```diff
@@ -27,6 +27,8 @@ gem 'pundit', '~> 2.1'
 gem 'puma', '~> 5.0'
 # Use SCSS for stylesheets
 gem 'sass-rails', '>= 6'
+# An utility first CSS framework
+gem 'tailwindcss-rails', '~> 0.3.0'
```
Comment (santib) [inline Gemfile:31]: I think the gem is not needed if we are going to use tailwind via webpacker, we just need the npm module, right?

Comment (santib) [reply Gemfile:31]: oh I guess you ran https://github.com/rails/tailwindcss-rails/pull/16/files#diff-5aa2349060350e9685318fe05e3403f899f77fb6e74fce0c6aee323aadbaf9b8 . I'm good then

**Review by santib — APPROVED** [review]
🎉

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -9,6 +9,7 @@
     <%= stylesheet_link_tag 'application', media: 'all', data: { turbo_track: :reload } %>
     <%= javascript_pack_tag 'application', data: { turbo_track: :reload } %>
     <%= yield :head %>
+    <%= stylesheet_pack_tag "application", "data-turbo-track": "reload" %>
```
Comment (JuannFerrari) [inline app/views/layouts/application.html.erb:12]: We could change this to:  `data: { turbo_track: :reload }`? to leave it the same as the others

**Review by JuannFerrari — APPROVED** [review]

**Review by lucasocon — APPROVED** [review]

File: postcss.config.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
```
Comment (jdrosales17) [inline postcss.config.js:1]: Just curious here, what is this `postcss` config for ?

**Review by jdrosales17 — APPROVED** [review]
🙌

**Comment by santib** [comment]
Just curious, but what are the disadvantages of adding tailwind via the asset pipeline (Rails default) ? Asking because now we have some CSS served by the asset pipeline and some others served by webpacker and sounds weird

**Comment by brunvez** [comment]
> Just curious, but what are the disadvantages of adding tailwind via the asset pipeline (Rails default) ? Asking because now we have some CSS served by the asset pipeline and some others served by webpacker and sounds weird

Decided to add this via webpacker now but change it in the near future

**Pull request merged**